server/repository/src/db_diesel/filter_sort_pagination.rs

```diff
@@ -61,6 +61,29 @@ impl EqualFilter<i32> {
     }
 }
 
+impl EqualFilter<f64> {
+    pub fn equal_to_f64(value: f64) -> Self {
+        EqualFilter {
```

Comment:
I've refactored to inline_init for this, see EqualFilter<i32>, if new filter s added less code to change